texastribune/donations
# Bump various front-end dependencies

#### What's this PR do?

Combines dependabot branches into one PR

#### Why are we doing this? How does it help us?

These are all attached to security patches

#### How should this be manually tested?

- `npm install` and `make restart`
- Click around the app and make sure all the JS features seem to work and there are no new console warnings
- Test the donations form
- Confirm you can log into the portal

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

N/A

#### What are the relevant tickets?

off-sprint - just general maintenance

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
